src/ui/flutter_app/lib/widgets/env_page.dart

```diff
@@ -21,49 +21,43 @@ import 'package:flutter/services.dart' show rootBundle;
 import 'package:sanmill/common/constants.dart';
 import 'package:sanmill/generated/l10n.dart';
 
-class EnvironmentVariablesPage extends StatefulWidget {
-  @override
-  _EnvironmentVariablesPageState createState() =>
-      _EnvironmentVariablesPageState();
-}
-
-class _EnvironmentVariablesPageState extends State<EnvironmentVariablesPage> {
-  String _data = "";
-
-  Future<void> _loadData() async {
-    final _loadedData =
-        await rootBundle.loadString(Constants.environmentVariablesFilename);
-    setState(() {
-      _data = _loadedData;
-    });
-  }
-
+class EnvironmentVariablesPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    _loadData();
-
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(S.of(context).environmentVariables),
-        centerTitle: true,
-      ),
-      body: ListView(
-        children: <Widget>[
-          Container(
-            padding: const EdgeInsets.only(
-              top: 16,
-              left: 16,
-              right: 16,
-              bottom: 16,
-            ),
-            child: Text(
-              _data != "" ? _data : 'Nothing to show',
-              style: const TextStyle(fontFamily: 'Monospace', fontSize: 12),
-              textAlign: TextAlign.left,
-            ),
+    return FutureBuilder<String>(
```

Comment:
Originally there were only 6 layers of nesting, after the refactoring, it is now increased to 9 layers.